index.html

```diff
@@ -224,6 +256,118 @@ <h5>
          </section>
       </div>
     </div>
+
+<!-- This is for desktop display -->
+<div class="flex-wrap">
+<div class="d-work">
+    <div class="proj-work-1 ">
+
+     <section class="proj-btn">
+         <button class="see">see project</button>
```

Comment:
- [ ] The title has a lot of space, when you are on a large screen you can see the issue much much better (see the red lines in the image below), please, fix it, try to align the title with the left side of the image with orange background below.

Yours | Figma design

![image](https://user-images.githubusercontent.com/60164239/130210881-1d3bbfcd-d229-435c-b14d-7c543c8268c4.png)